src/main/webapp/WEB-INF/views/Hello123.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
```

Comment:
Sofa will make you to remove this jsp :)

Reply:
done